core/src/main/resources/schema/infinispan-config-14.0.xsd

```diff
@@ -968,6 +968,21 @@
           </xs:documentation>
         </xs:annotation>
       </xs:element>
+      <xs:element name="query" minOccurs="0">
+        <xs:annotation>
+          <xs:documentation>
+            Defines query options for cache
+          </xs:documentation>
+        </xs:annotation>
+        <xs:attribute name="default-max-result" type="xs:integer" default="${Query.default-max-result}">
+          <xs:annotation>
+            <xs:documentation>
+              Applies the given value as maxResults as a default to any query (indexed, non-indexed and hybrid).
```

Comment:
A suggestion:
Limits the number of returned results by a query. Applies to indexed, non-indexed, and hybrid queries. Setting the default maxResult significantly improves performance of queries that don't have an explicit maxResults set.

Reply:
Looks good to me